Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-locationviewlocationstabview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Location View Locations Tab view class.
+     *
+     * @module ez-locationviewlocationstabview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Location View Locations Tab View class.
+     *
+     * @namespace eZ
+     * @class LocationViewLocationsTabView
+     * @constructor
+     * @extends eZ.LocationViewTabView
+     */
+    Y.eZ.LocationViewLocationsTabView = Y.Base.create('locationViewLocationsTabView', Y.eZ.LocationViewTabView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._fireLoadLocations;
+
+            this.after('locationsChange', function (e) {
```

Comment:
you set the property `_watchAttribute` instead (see AsynchronousView code/comments)